src/main.module.ts

```diff
@@ -42,10 +42,16 @@ import { LoggingInterceptor } from './logging/logging-interceptor.helper';
             clientRequest: {
               logKey: 'req',
               headersLogKey: 'headers',
+              bodyLogKey: 'data',
```

Comment:
Should we use constant BODY_LOG_KEY here?

Reply:
Yep, good spot :)